frontend/src/employee-frontend/components/unit/TabGroups.tsx

```diff
@@ -1,3 +1,13 @@
+import { renderResult } from '../async-rendering'
+import React, { Dispatch, SetStateAction, useContext, useMemo } from 'react'
+import { ContentArea } from 'lib-components/layout/Container'
+import { UnitContext } from '../../state/unit'
+import { FixedSpaceColumn } from 'lib-components/layout/flex-helpers'
+import Groups from '../../components/unit/tab-groups/Groups'
+import MissingGroupPlacements from '../../components/unit/tab-groups/MissingGroupPlacements'
+import { combine, Result } from 'lib-common/api'
+import { Action } from 'lib-common/generated/action'
+
 {
   /*
 SPDX-FileCopyrightText: 2017-2020 City of Espoo
```

Comment:
Move the licence stuff to the top.

Reply:
Good catch! Fixed license comment style in all .tsx files now.